tests/aspnetcore5/config.json

```diff
@@ -18,5 +18,6 @@
         "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=0",
         "DOTNET_CLI_TELEMETRY_OPTOUT=1"
     ],
+    "ThreadStackSize": "16m",
```

Comment:
16M is quite high, considering dotnet runtime can create a lot of threads. Is the minimum size which works?

Reply:
I remember trying out different value for aspnet sample here, and 16m is where it won't crash/hang